doctr/models/recognition/crnn/pytorch.py

```diff
@@ -194,6 +194,9 @@ def forward(
         return_preds: bool = False,
     ) -> Dict[str, Any]:
 
+        if self.training and target is None:
+            raise ValueError('Need to provide labels during training for teacher forcing')
```

Comment:
I think the message should be different here. Teacher forcing is specific to the SAR :sweat_smile:

Reply:
Oh yes i agree without a similar (bigger) model or RNN/LSTM this makes no sense ..